src/tools/Erase.ts

```diff
@@ -1,37 +1,58 @@
 import { Tool } from '@/tools/Tool'
 import Konva from 'konva'
 import { CanvasElement } from '@/types/Canvas'
-import uuid from 'uuid'
 
 export default class Erase implements Tool {
   // eslint-disable-next-line no-useless-constructor
   constructor (public readonly name: string) {
   }
-  // eslint-disable-next-line
-  mouseDownAction = (e: Konva.KonvaPointerEvent, CanvasElement: CanvasElement, layer: Konva.Layer, socket: WebSocket): void => {
 
+  mouseDownAction = (e: Konva.KonvaPointerEvent, canvasElement: CanvasElement, layer: Konva.Layer, socket: WebSocket): void => {
+    this.findAndEraseGroup(e, canvasElement, layer, socket)
   }
-  // eslint-disable-next-line
-  mouseMoveAction = (e: Konva.KonvaPointerEvent, CanvasElement: CanvasElement, layer: Konva.Layer, socket: WebSocket): void => {
 
+  mouseMoveAction = (e: Konva.KonvaPointerEvent, canvasElement: CanvasElement, layer: Konva.Layer, socket: WebSocket): void => {
+    this.findAndEraseGroup(e, canvasElement, layer, socket)
   }
   // eslint-disable-next-line
-  mouseUpAction = (e: Konva.KonvaPointerEvent, CanvasElement: CanvasElement, layer: Konva.Layer, socket: WebSocket): void => {
+  mouseUpAction = (e: Konva.KonvaPointerEvent, canvasElement: CanvasElement, layer: Konva.Layer, socket: WebSocket): void => {
 
   }
 
-  // eslint-disable-next-line
+  findAndEraseGroup = (e: Konva.KonvaPointerEvent, canvasElement: CanvasElement, layer: Konva.Layer, socket: WebSocket): void => {
+    const groupId = e.target.parent?.attrs.id
+    this.eraseGroup(canvasElement, layer, groupId)
+    this.sendToWebSocket(canvasElement, socket)
+  }
+
+  eraseGroup = (canvasElement: CanvasElement, layer: Konva.Layer, groupId?: string): void => {
+    if (groupId !== undefined && groupId !== null) {
+      const group = layer.getChildren(node => {
+        return node.attrs.id === groupId
```

Comment:
Do not need separate lines for return. Remove curly braces and return keyword then put on one Line